README.rst

```diff
@@ -11,6 +11,8 @@ Usage
 
 In your ``Dockerfile``, put the following code at the end::
 
+    ENV DEFAULT_KERNEL <your kernel>  # set to python3 or ir
```

Comment:
In this and the example Dockerfile, we should have been setting this environment variable earlier to avoid an error when running `civis-jupyter-notebooks-install`.

Reply:
Just that `DEFAULT_KERNEL` wasn't defined.

Reply:
ah, that makes sense